classes/class-wc-connect-account-settings.php

```diff
@@ -39,21 +39,23 @@ public function get() {
 		$last_box_id                 = $last_box_id === 'individual' ? '' : $last_box_id;
 		$last_service_id             = get_user_meta( get_current_user_id(), 'wc_connect_last_service_id', true );
 		$last_carrier_id             = get_user_meta( get_current_user_id(), 'wc_connect_last_carrier_id', true );
+		$wcshipping_migration_state  = WC_Connect_Options::get_option( 'wcshipping_migration_state' );
 
 		return array(
 			'storeOptions' => $this->settings_store->get_store_options(),
 			'formData'     => $this->settings_store->get_account_settings(),
 			'formMeta'     => array(
-				'can_manage_payments'     => $this->settings_store->can_user_manage_payment_methods(),
-				'can_edit_settings'       => true,
-				'master_user_name'        => $connection_owner ? $connection_owner->display_name : '',
-				'master_user_login'       => $connection_owner ? $connection_owner->user_login : '',
-				'master_user_wpcom_login' => $connection_owner_wpcom_data ? $connection_owner_wpcom_data['login'] : '',
-				'master_user_email'       => $connection_owner_wpcom_data ? $connection_owner_wpcom_data['email'] : '',
-				'payment_methods'         => $this->payment_methods_store->get_payment_methods(),
-				'add_payment_method_url'  => $this->payment_methods_store->get_add_payment_method_url(),
-				'warnings'                => array( 'payment_methods' => $payment_methods_warning ),
-				'is_eligible_to_migrate'  => $this->settings_store->is_eligible_for_migration()
+				'can_manage_payments'        => $this->settings_store->can_user_manage_payment_methods(),
+				'can_edit_settings'          => true,
+				'master_user_name'           => $connection_owner ? $connection_owner->display_name : '',
+				'master_user_login'          => $connection_owner ? $connection_owner->user_login : '',
+				'master_user_wpcom_login'    => $connection_owner_wpcom_data ? $connection_owner_wpcom_data['login'] : '',
+				'master_user_email'          => $connection_owner_wpcom_data ? $connection_owner_wpcom_data['email'] : '',
+				'payment_methods'            => $this->payment_methods_store->get_payment_methods(),
+				'add_payment_method_url'     => $this->payment_methods_store->get_add_payment_method_url(),
+				'warnings'                   => array( 'payment_methods' => $payment_methods_warning ),
+				'is_eligible_to_migrate'     => $this->settings_store->is_eligible_for_migration(),
+				'wcshipping_migration_state' => $wcshipping_migration_state,
```

Comment:
I considered having the React app to make an API `GET` call to retrieve this value from `useEffect()`. But we already pass `is_eligible_to_migrate` in, so it makes sense to me to pass `wcshipping_migration_state` in as well.